common/src/main/java/bisq/common/monetary/PriceQuote.java

```diff
@@ -46,7 +46,7 @@
 @Slf4j
 public final class PriceQuote implements Comparable<PriceQuote>, PersistableProto {
     @Setter
-    private static final String QUOTE_SEPARATOR = "/";
+    private static String QUOTE_SEPARATOR = "/";
```

Comment:
Why not remove the Setter? I cannot see any usage for setting and also it would be weird why we should set that from outside.

Reply:
I did the small change so it gets ready for merge